br-webui/_includes/header.liquid

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
why is this necessary?

Reply:
I am not exactly sure but it was for the toggle switch. This line was there in one of the styles i found. I will remove it and see how its affecting anything.